pcap_ex.c

```diff
@@ -290,7 +290,7 @@ pcap_ex_compile_nopcap(int snaplen, int dlt, struct bpf_program *fp, char *str,
 	char ebuf[PCAP_ERRBUF_SIZE];
 	int ret = -1;
 	
-	_mktemp(path);
+	mktemp(path);
```

Comment:
So what's the "macOS issue" here?

On *all* UN\*Xes - Linux, macOS, Solaris, \*BSD, etc. - you have `mktemp()` and `unlink()`.  `_mktemp()` and `_unlink()` are Windows-isms, avoiding polluting the C namespace with UN\*X names.

If the goal is to work on UN\*X and Windows, you probably want to arrange to use `mktemp()` and `unlink()` on UN\*X and `_mktemp()` and `_unlink()` on Windows.